### PR TITLE
removed the heading-underlined class

### PR DESCRIPTION
**Pull request by Sandravaphilips**
Loom https://www.loom.com/share/64a491cc03f741db89e3d4751b32f132?sid=4c6cb653-6737-4706-894d-e4e69a50354e

### CODE REVIEW

File: africanlii/templates/africanlii/au_detail_page.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <div class="container">
       {% block breadcrumbs %}{% endblock %}
       {% block page-heading %}
-        <h1 class="my-4 heading-underlined">{% trans 'African Union (AU)' %}</h1>
+        <h1 class="my-4 heading">{% trans 'African Union (AU)' %}</h1>
```
Comment (longhotsummer) [inline africanlii/templates/africanlii/au_detail_page.html:11]: what is `.heading`? can that be removed?

**Comment by longhotsummer** [comment]
image

the spacing here is off. The "ECOWAS court" is closer to "recent judgments" than the  judgments are to "recent judgments".

the heading and its logically connected content should have less space between them, than the previous heading and its content.

This looks better, where the court grouping heading and the items in that group have (slightly) less whitespace between them, than the groups themselves.

image

**Comment by longhotsummer** [comment]
can you provide new screenshots, pls?

**Comment by Sandravaphilips** [comment]
@longhotsummer 
Screenshot 2024-01-31 at 14 18 22

**Review by longhotsummer — APPROVED** [review]

**Pull request merged**